rerun_py/tests/unit/test_points2d.py

```diff
@@ -0,0 +1,263 @@
+from __future__ import annotations
+
+import itertools
+
+import numpy as np
+import rerun2 as rr
+from rerun2 import components as rrc
+
+# NOTE: uncomment these to get a better auto-completion experience...
+# from rerun_sdk import rerun2 as rr
+# from rerun_sdk.rerun2 import components as rrc
```

Comment:
oof. hopefully soon gone once we no longer need the rerun2 dance?

Reply:
Yeah this'll go away when backporting.

Though, again, I still have absolutely no clue why half the tools find it, and the other half don't...